### DISCUSSION TITLE
Gantt chart example from the medium article

### DISCUSSION THREAD

**Discussion by Mistyputt** [airbnb/visx#1358] [Q&A]
In the medium article [Introducing visx from Airbnb](https://medium.com/airbnb-engineering/introducing-visx-from-airbnb-fd6155ac4658) a screenshot of a dependency-inclusive gantt chart for data pipeline task sequence analysis is shown. Does anyone know if the source code for that example has been made available in the gallery or in a CodeSandbox or similar?

<img width="690" alt="image" src="https://user-images.githubusercontent.com/5863950/138091735-7c4df8df-e4f3-4ff8-a39e-606576ab284d.png">

Would be a great starting point I think.

**Comment by williaster** [comment]
Hey @Mistyputt 👋  thanks for checking out `visx`. This hasn't been published anywhere yet, no. are you interested in just the gantt chart, or also the dependency aspect? 

It's pretty straight forward either way so I can try to create a sandbox to stick in the gallery or post here. (it's more straight forward without the arcs ... they are more complicated because they are responsive in our tool so have to know about row height)

**Reply by Mistyputt** [reply]
I'm interested in both actually so if you basically have the pieces to put it together in a sandbox it would be fantastic. I guess the dependency visualization introduces a bit more complexity for those who "just" want a timeline so maybe others would enjoy a separate "timeline only".

Since posting I have been playing around with the timeline from this https://codesandbox.io/s/71znlx7551 (found it here https://spectrum.chat/vx/general/gantt-visualization-with-vx~0121f2dd-d0e2-4256-bd56-5f1e04443631) and the arc usage from https://codesandbox.io/s/color-wheel-cpxpz?file=/index.tsx  (found it in #886). I see that arc's take a bit of calculation to get right (position the origin with a group and controlling the size with inner/outerRadius)

I'm still learning how to control the various components so a sandbox of that demo would speed me up considerably! 

Thanks for making this library, I really enjoy the experience using it so far and I like your "here's the components, build your own chart" approach.
